ci/scripts/tests/test_rocotostat.py

```diff
@@ -0,0 +1,81 @@
+import sys
+import os
+from shutil import rmtree
+
+script_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.join(os.path.dirname(script_dir), 'utils'))
+
+from rocotostat import rocoto_statcount, rocotostat_summary, is_done, is_stalled, CommandNotFoundError
+from wxflow import which
+
+test_data_url = 'https://noaa-nws-global-pds.s3.amazonaws.com/data/CI/'
+
+testdata_path = 'testdata/rocotostat'
+testdata_full_path = os.path.join(script_dir, testdata_path)
+
+wget = which('wget')
+if not os.path.isfile(os.path.join(testdata_full_path, 'database.db')):
+    wget.add_default_arg([
+        '-P', testdata_full_path, test_data_url + str(testdata_path) + '/workflow.xml',
+        test_data_url + str(testdata_path) + '/database.db'])
+    wget()
```

Comment:
Why go through all of this, when you can;
```python3
import wget
URL = test_data_url + str(testdata_path) + '/workflow.xml'
wget.download(url)
```

Have `wget` pip installed in the `ci-unit_tests.yaml`

Reply:
"go through all of this"?  Looks pretty strait forward to me, but ok.

Reply:
I think you might be confused on what the arguments are for.